subworkflows/nf-core/differential_functional_enrichment/tests/all.config

```diff
@@ -0,0 +1,64 @@
+process {
+
+    cpus = 1
+
+    withName: 'DESEQ2_DIFFERENTIAL' {
+        ext.args = { [
+            "--round_digits 5",
+            "--vs_method rlog",
+            (meta.blocking == null) ? "" : "--blocking_variables $meta.blocking"
```

Comment:
should this line be part of the module?

Reply:
We're not supposed to assume meta fields in modules.